### ISSUE TITLE
Formalize integration of new crowd research cohort with the existing Collective via a Crowd Research PI operational group

### ISSUE THREAD

**Issue by mbernst** [crowdresearch/collective#12]
We have been planning to onboard a new cohort of contributors in the coming weeks, to begin in January. While the Collective's governance process is consensus-based, Crowd Research process has previously been led by the PI faculty member, with the PI guiding research direction. How do we integrate these two?

The problems we need to solve:
- Does the new cohort immediately gain full rights to break consensus and create proposals?
- How do we clarify the role the PI has?

### My proposal
I am inspired by platforms such as StackOverflow and Discourse, which give newcomers “beginner" rights that expand over time. On StackOverflow, at first you can ask questions, but you can’t upvote questions as useful until you prove yourself. Specifically, I would propose that the new cohort be able to view proposals and create proposals and vote, but not be able to break consensus until they have been active for a period of time (e.g., two months).

This would produce an onboarding path for newcomers that looks like: (1) newbie: join and start participating —> (2) regular member: gain full rights in the consensus process —> (3) advanced member: be elected to an operational or quality review group

Since many of the volunteers join the cohort to work with a PI and get recommendation letters, it seems like we need to ensure that they are given ample opportunity to do so. The proposal on the table — though I am very open to alternatives — would be to have a "Crowd Research PI" operational group (initially, a single role initialized to @mbernst) that is responsible for the crowd research process. This group is responsible for setting milestones, giving feedback on research directions, mentoring the papers, and so on. The process would be collaborative with the rest of the Collective, but the PI is empowered to make those decisions. The PI would have no special abilities relative to Daemo governance: for example, any changes to the live platform or its policies would require the usual consensus process. The intention would be for crowd research to work in concert with the Collective throughout, so that research ideas can be tested, but the researchers would likely lead any development needed to run those tests.

---
**Use comments to share your response** or use emoji 👍 to show your support. **To officially join in**, add yourself as an assignee to the proposal. **To break consensus**, comment using [this template](https://raw.githubusercontent.com/crowdresearch/collective/master/docs/breaking_consensus). To find out more about this process, read the [how-to](https://github.com/crowdresearch/collective/blob/master/docs/CONTRIBUTING.md).

**Comment by iceLearn** [comment]
I see this as a great proposal however, I have some questions, so would like to get more clarifications .

1. I like the idea of restricting new cohort  based on the active participation ( eg for 2 months )
 
"_new cohort be able to view proposals and create proposals and vote, but not be able to break consensus until they have been active for a period of time (e.g., two months)."_

If they assigned voting right it could be easily get approved as there is always a possibility of higher voting from them since we have less active  regular and advance members.  
Therefore `I would suggest to eliminate voting right until they become regular members.`  But of course new ideas and proposals are welcome. And breaking conses also will give an insight for us to see the feasibility, but voting by only regular and advance members. 

2. "Crowd Research PI" operational group -> 
 
It is understandable that any cohort who will be joining this group may have intentions to get research experience and eventually recommendations. I support the Crowd Research PI ideas, but as @mbernst  can not fully function the research process with new cohort, you are proposing a group to assist you in this initiative by setting milestones, giving feedback on research directions, mentoring the papers, and so on.. how do you decide the group? What are the criteria for that selection?

**Comment by mbernst** [comment]
Great questions. So you suggest that they can view proposals and create proposals, but not break consensus or vote? Or view and create proposals and break consensus, but not vote?

With respect to the operational group, I can see two ways that we might move forward. One would be just to have an operational group for the PI(s): specifically empowered to create or approve milestones, give feedback on research, mentor papers. And others in the group can support as much as they want and can. Another approach would be to create an operational group that's more than just the PI, where the group together takes responsibility for those things.

Also, there may be some option that does not involve this role, or that manages the research process collectively. I am open to those ideas...I couldn't come up with something reasonable.

**Comment by iceLearn** [comment]
On new cohort - > I was suggesting for them to view/ create / break consensus but not vote. 
( because  new ideas should welcome at the same time among them can have different point of view which is good to analyse the proposals.)

On the  operational group - 
First and second approach both seems good. But I like the second approach so there will be a strategic plan discussion  among the group  about creating milestones, research agenda and then executing with the new cohort.

**Comment by neilthemathguy** [comment]
This is interesting. I can help with this in upcoming weeks. **It is important to highlight that as the members from new cohort join Crowd Research Collective, they become a part of Daemo.**

**Comment by shirishgoyal** [comment]
I fully agree with @iceLearn that new cohort should be allowed to create new proposals as well as allowed to contest consensus. But their voting rights can be limited for initial 1-2 months when they are still grasping what their role and responsibilities entail. 

In addition, I think it is very important to transform research operational group (whatever form it might take) to act in a more advisory role and provide constructive feedback on how research ideas can be improved than defining research direction to take (as it should be kept under collective decision making process). There should not exist a conflict of interest which bars the collective to pursue any research idea it wants to take forward.

**Comment by mbernst** [comment]
I want to make sure I've synthesized this all correctly: @shirishgoyal and @iceLearn both suggest that the new cohort can create proposals and break consensus from the beginning, but voting rights are only added after some tenure has accrued. We need to define what this means. Is it "active time"? Or "time after being added to the Collective"? I assume the latter, which is easier to track. And if time, how long? I have heard "two months".

@neilthemathguy wants to make sure that in onboarding we make clear that they are joining the Collective, not just a Crowd Research project. I agree we need to make this clear.

@shirishgoyal suggests that the ops group gives research advice but does not "define the research direction to take". This is related to what I was trying to go for with "The PI would have no special abilities relative to Daemo governance: for example, any changes to the live platform or its policies would require the usual consensus process." I like @shirishgoyal's suggestion in that it indicates a tight connection between SRCC (I just acronym'ed it: Stanford Crowd Research Collective), research and Daemo: that Daemo is a research platform doing research as it grows.

Please make sure I've synthesized this correctly.

**Comment by mbernst** [comment]
# Summary
We have agreed that we should create an operational group for Crowd Research PI, initialized to @mbernst. 

We have agreed that there will be an onboarding path for newcomers. The high level framework of this is that newcomers will have the ability to create proposals and break consensus, but not have voting rights initially. There will be a period of time or activity level required to obtain this full membership. Details of this requirement will be hashed out in a future proposal.

We have further agreed that the PI will give research feedback but does not have unilateral control over the research direction. Specifics about this line will be hashed out in a future proposal.

----
As per discussion today, the goal moving forward will be to have this summary prior to the weekly meeting, so everyone knows what has been agreed upon.
